.gitignore

```diff
@@ -9,5 +9,8 @@ __pycache__
 uploads/
 htmlcov
 .venv
+localhost/
+teams.xlsx
+application/create_accounts.py
```

Comment:
If you have local changes that you don't want to commit, you don't have to add it to the .gitignore which is meant to be used for everyone developing on the project. Instead you can use `.git/info/exclude` to specify your personal excludes.

Reply:
👍